src/Collection/AuditedCollection.php

```diff
@@ -187,7 +187,7 @@ public function containsKey($key)
     }
 
     /**
-     * @return object
+     * @return object|null
      *
      * @phpstan-return T
```

Comment:
If you modify the above, this one should be changed too right?

Reply:
It was an error